exam_1/task1/main.c

```diff
@@ -0,0 +1,99 @@
+#include <stdio.h>
+#include <string.h>
+
+int tripleTransformation(char *binary, char *octale)
```

Comment:
```suggestion
int triadTransformation(char *binary, char *octale) //BitBlockTransformation
```
"Тройное преобразование"?
Лучше "преобразование триады" или "преобразование блока бит"
(-0.25)